shell_integration/dolphin/CMakeLists.txt

```diff
@@ -43,6 +40,8 @@ add_library(${OWNCLOUDDOLPHINHELPER} SHARED ownclouddolphinpluginhelper.cpp)
 target_link_libraries(${OWNCLOUDDOLPHINHELPER} Qt5::Network Qt5::Gui)
 generate_export_header(${OWNCLOUDDOLPHINHELPER} BASE_NAME ownclouddolphinpluginhelper)
 install(TARGETS ${OWNCLOUDDOLPHINHELPER} LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
+target_include_directories(${OWNCLOUDDOLPHINHELPER} PUBLIC ${CMAKE_CURRENT_BINARY_DIR} ${PROJECT_BINARY_DIR} ${PROJECT_BINARY_DIR}/src)
+target_compile_definitions(${OWNCLOUDDOLPHINHELPER} PUBLIC -DQT_USE_FAST_CONCATENATION -DQT_USE_FAST_OPERATOR_PLUS)
```

Comment:
Private

Reply:
I could only make them private when repeating this code for the other two targets.

Reply:
Please do so, setting compile flags is not duplicating code and such build flags should never be propagated to all dependent targets.